src/app/components/job-list-item/job-list-item.component.html

```diff
@@ -1,20 +1,22 @@
 <div class="job-list-container" (click)="onListItemClick()">
-  <p class="job-list-container-header">{{job.name}}</p>
-  <p class="adress-field" >
-    <i class="fa fa-map-marker" aria-hidden="true"></i> {{job.address}}
-  </p>
+  <div class="date-container col-xs-4">
+    {{job.jobDate | date: 'MMM'}}<br/>
+    {{job.jobDate | date: 'dd'}}
+  </div>
 
-  <div class="row">
-    <div class="col-xs-8">
-      <span class="total-salary">{{job.grossAmount}} {{job.hourlyPay.currency}}</span>
-      <span class="salary-per-hour">
-        {{job.hours}}h ({{job.hourlyPay.grossSalary}} {{job.hourlyPay.currency}}/h)
-      </span>
-    </div>
+  <div class="job-meta-details">
+    <p class="job-list-container-header">{{job.name}}</p>
+    <p class="adress-field" >
+      <i class="fa fa-map-marker" aria-hidden="true"></i>  {{job.address}}
+    </p>
 
-    <div class="salary-per-hour col-xs-4">
-      {{job.jobDate | date: 'yyyy-MM-dd'}}
+    <div class="row">
+      <div> <span class="total-salary">{{job.grossAmount}} {{job.hourlyPay.currency}}</span>
```

Comment:
Change `{{job.grossAmount}} {{job.hourlyPay.currency}}`
 to `{{job.grossAmountWithCurrency}}`